### PR TITLE
BAQE-2455 add dynamic parameter passing for kogito sources-zip variants

### PR DESCRIPTION
**Pull request by jstastny-cz**
* Adding dynamic parameters passing from overall maven build into invoked projects for sources-zip variants of kogito-* projects.
* Reducing existing list of passed properties to the supported subset (overriding platform BOM GAVs are not applicable).

### CODE REVIEW

File: run-tests-with-build-kogito-examples-parent/kogito-examples-scm-checkout-test/pom.xml [hunk 1]
```diff
@@ -26,7 +26,11 @@
       kogito.bom.version,
       optaplanner.bom.group-id,
       optaplanner.bom.artifact-id,
-      optaplanner.bom.version
+      optaplanner.bom.version,
```
Comment (radtriste) [inline run-tests-with-build-kogito-examples-parent/kogito-examples-scm-checkout-test/pom.xml:29]: do we still need the optaplanner here ?
AFAIK, optaplanner is out of kogito examples

File: run-tests-with-build-kogito-examples-parent/kogito-examples-sources-zip-test/pom.xml [hunk 2]
```diff
@@ -12,6 +12,28 @@
...
+      quarkus.platform.group-id,
+      quarkus.platform.artifact-id,
+      quarkus.platform.version,
+      quarkus-plugin.version,
+      kogito.bom.group-id,
+      kogito.bom.artifact-id,
+      kogito.bom.version,
+      optaplanner.bom.group-id,
+      optaplanner.bom.artifact-id,
+      optaplanner.bom.version,
```
Comment (radtriste) [inline run-tests-with-build-kogito-examples-parent/kogito-examples-sources-zip-test/pom.xml:29]: same applies here

**Review by radtriste — APPROVED** [review]
Minor suggestion

**Comment by radtriste** [comment]
tested ok on 1.25 CR3 build

**Review by MarianMacik — APPROVED** [review]
Looks good!

**Pull request merged**